draft-ietf-scone-protocol.md

```diff
@@ -345,6 +348,41 @@ and the corresponding bitrate for each.
 {: #ex-rates title="Examples of SCONE signals and corresponding rates"}
 
 
+
+## Monitoring Period {#time}
+
+The time over which throughput advice applies is defined to be
+a period of 67 seconds.
+
+Protocol participants can use a different period,
+depending on their role.
+Senders can limit their send rate over any time period
+up to 67 seconds.
+Network elements can monitor and apply limits to send rates
+using time period of at least 67 seconds.
+
+The choice of 67 seconds is a compromise between competing interests.
+Longer periods allow applications more flexibility
+in terms of how to allocate bandwidth over time.
+Shorter periods allow networks to administer policies more tightly.
+Also, when circumstances change,
+applications are better able recover
+without exceeding limits for a significant time
+if they have exceeded limits.
+
+The choice of 67 seconds, as a prime number,
+also ensures that problems in tuning applications
```

Comment:
Does it "ensure" this (guarantee)? I wonder if it mitigates potential problems....

Reply:
That might be a bit too strong language, yes.